doc/nrf/ug_zigbee_memory.rst

```diff
@@ -106,12 +107,14 @@ The following tables list memory requirement values (in KB) for Zigbee samples.
 Stack memory requirements
 *************************
 
-The following table lists stack memory requirement values (in KB) for ``main`` and ``zboss`` threads.
+The following table lists stack memory requirement values for ``main`` and ``zboss`` threads.
```

Comment:
I suggest using units to be crystal clear:
The following table lists stack memory requirement values (in KB) for ``main`` and ``zboss`` threads.

Reply:
Not sure I follow. The unit reference was moved two lines down, with additional explanation.

Reply:
Discussed on Teams.